stories/core/Highlighter.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { number, text } from '@storybook/addon-knobs'
+import { Highlighter } from 'react-ui-core/src'
+import StoryBookTheme from '../theme/Storybook.css'
+
+export const DefaultHighlighter = (
+  <div>
+    <Highlighter
+      className={StoryBookTheme['Story-padding']}
+      pattern="test"
+    >
+      this is a test to see if this works, test seems to have passed the test
+    </Highlighter>
+  </div>
+)
+
+export const HighlighterWithIndex = () => {
```

Comment:
This story is broken.

Reply:
Let me take a look at why the knobs aren't working

Reply:
Fixed now